### ISSUE TITLE
Springer link does not include ISSN

### ISSUE THREAD

**Issue by rugk** [Langenscheiss/bibitnow#31]
When you cite an entry from Springer link `@article`, the citation is almost perfect, but it misses the ISSN entry.

You can manually download the RIS citation with a link from Springer and I assume this add-on also actually does that.
The `SN` field there contains the , which [this converter](https://www.bruot.org/ris2bib/) converts correctly.

**Comment by Langenscheiss** [comment]
Of course bibitnow already has an internal RIS converter.
And on this link,

https://link.springer.com/article/10.1007/s11238-019-09742-3

the ISSN shows up perfectly here. Could you send a link to where it fails?

**Comment by Langenscheiss** [comment]
I have tested SpringerLink and also looked at the code of the site adjuster. Since you claim you could download a ris with the issn, I also checked whether the dynamic citation download works on SpringerLink and it does perfectly on my end. Sometimes, I decide to discard certain information from the downloaded ris if it turns out to be unreliable, but in case of Springer, this is not the case as I also explicitly confirmed. There is an SN entry in the ris file, and bibitnow does extract it on my end. Furthermore, Springer provides the ISSN also in the static meta data. So the only way this could have happened is that this particular journal on SrpingerLink you wanted does not provide the issn information as usual in the website html, and for some reason the dynamic citation download also did not work (two strange things, but not impossible)
In other words, I can only work on this is if you provide the link. Otherwise, I cannot reproduce this issue.

**Comment by rugk** [comment]
Hmm sorry, dunno. Cannot find it right now anymore. Maybe it actually worked and I overlooked it.
Nevermind, thanks for the investigation anyway…

**Comment by Langenscheiss** [comment]
Cool. No problem. It's good to have checked it anyhow. Thanks

In general, it helps to always post a link. Even if it seems like a general issue, it might be something site specific. If you happen to find it, you can open another issue and I will look at it :)
